src/QDL/qdl.js

```diff
@@ -258,17 +224,18 @@ export function useFastboot() {
         async function unpackImages() {
           for await (const [image, onProgress] of withProgress(manifest.current, setProgress)) {
             setMessage(`Unpacking ${image.name}`)
+            if (image.name == "system") continue;
```

Comment:
what's this for?

Reply:
my bad, it was for testing, forgot to delete, will update